### PR TITLE
Add Kotlin starter bot

### PR DESCRIPTION
**Pull request by juan-db**

### CODE REVIEW

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		}
+		if (!ip.startsWith("http://")) {
+			ip = "http://$ip"
+		}
+		return "$ip:5000/runnerhub"
+	}
+
+	@JvmStatic
+	fun main(args: Array<String>) {
+		val token = System.getenv("TOKEN") ?: UUID.randomUUID().toString()
```
Comment (QuattroViper) [inline starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt:28]: The correct System Environment variable is REGISTRATION_TOKEN. 
If the token is not correct, we cannot update the bot's score and placement.

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+			hubConnection.on("ReceiveGameComplete", {
+				println("Game complete: $it")
+			}, String::class.java)
+
+			hubConnection.start().blockingAwait()
+			println("Connection established with runner.")
+
+			Thread.sleep(1000)
+			hubConnection.send("Register", token, "KotlinBot")
```
Comment (QuattroViper) [inline starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt:64]: Can we move the "KotlinBot" to a variable in the beginning of the class called botNickname, to clearly state that people can change their bot's nickname.

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt (hunk 1 above)
Comment (juan-db) [reply starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt:28]: Am I misunderstanding something or is the Java starter bot also wrong then?
https://github.com/EntelectChallenge/2021-Galaxio/blob/b5b0f51e3fb4854fdc9d3f4104e3ec60e86b5f4e/starter-bots/JavaBot/src/main/java/Main.java#L15

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt (hunk 2 above)
Comment (juan-db) [reply starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt]: Fixed in 5ebc13e3ccf4c46df304953d789d3a684c9bd7dc.

File: starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt (hunk 1 above)
Comment (juan-db) [reply starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt:29]: The documentation also seems to match the Java bot.
https://github.com/EntelectChallenge/2021-Galaxio/blob/develop/building-a-bot.md#21-get-your-token-from-your-environment

Comment (ThePheonixGuy) [reply starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt:29]: The token must be the Pascal Case `Token` or alternatively `REGISTRATION_TOKEN`.

I will update the documentation to correct this, apologies. But please update here

Comment (juan-db) [reply starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt]: Thanks for the clarification. :slightly_smiling_face: 

Fixed in 57e6ad502636b30e2246b5fa8d84b255e6c5a5d4.

**Pull request merged**